src/bundle/DependencyInjection/Configuration/Parser/Notifications.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\DependencyInjection\Configuration\Parser;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\AbstractParser;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+/**
+ * Configuration parser for subtree related operations.
+ *
+ * Example configuration:
+ * ```yaml
+ * ezpublish:
+ *   system:
+ *      admin_group: # configuration per siteaccess or siteaccess group
+ *          notifications:
+ *              warning: # type of notification
+ *                  timeout: 5000 # in milliseconds
```

Comment:
`miliseconds`

Reply:
`milliseconds` is correct - https://en.wikipedia.org/wiki/Millisecond